docs/quick-start-docker-cluster.md

```diff
@@ -0,0 +1,165 @@
+# Quick start for docker-cluster
+
+The docker-cluster image is similar to docker-standard, but with the following changes and features:
+
+* Cluster of 4 metrictanks (with 2x replication)
+* Separate Graphite monitoring server
+* The metrictank binary is **not** baked in
+* Loads custom configurations and scripts
+* Supports tags
+* More dashboards are available
+* Kafka input and clustering backend
+* Short chunkspan & numchunks (benefits: easy to trigger mem and mem_and_cass requests, frequent cass saves, notifier messages etc)
+* Prometheus monitoring
+* Jaeger tracing
+* caddy-proxied endpooints/datasources to check with different org-id auth headers
+* tsdb-gw
+
+
+The following programs are required to build and run the docker-cluster image:
+
+* [Docker](https://docs.docker.com/install/)
+* [docker-compose](https://docs.docker.com/compose) >= version 1.6
+* [Go](https://golang.org/doc/install) (needed to build metrictank)
+
+## Getting the repository
+
+Pull down the repository and its dependencies: 
+
+```
+go get github.com/grafana/metrictank/...
+cd $GOPATH/src/github.com/grafana/metrictank
+```
+
+## Build metrictank
+
+Build metrictank and the docker images:
+```
+make
+```
+
+`make` creates the folder `$GOPATH/src/github.com/grafana/metrictank/build`. If you experience a permissions error during `make` it is probably related to this. Try either changing ownership (`chown`) or permissions (`chmod`) on the folder to something your current user is able to read and write, then run `make` again.
+
+You may receive a few build errors at the end relating to QA, ignore them. Ensure the binary `$GOPATH/src/github.com/grafana/metrictank/build/metrictank` was built.
```

Comment:
what build errors?

Reply:
It was a `PATH` problem for me. During `go generate` the scripts were not finding `msgp` and `stringer`. It is fixed now. I would like to add a note though in case other people encounter it.

Reply:
Yes, `make qa` was calling go-generate.sh which then called `go generate`. `msgp` and `stringer` were being installed into `$GOPATH/bin`, but that was not in my `$PATH` so `msgp` and `stringer` could not be run causing `make qa` to fail. After I added `$GOPATH/bin` to `$PATH` it all worked without issue.

Reply:
i see. to solve that, i think whenever any of our docs tell people to install go, we should extend that to include the PATH change.
that should remove this class of errors.

Reply:
I agree, I'll add that